Adyen/Assets/zh-TW.lproj/Localizable.strings

```diff
@@ -31,7 +32,7 @@
 "adyen.card.stored.expires" = "%@ 到期";
 "adyen.dropIn.stored.title" = "確認 %@ 付款";
 "adyen.dropIn.preselected.openAll.title" = "變更付款方式";
-"adyen.continueTo" = "繼續前往%@";
+"adyen.continueTo" = "繼續前往 %@";
```

Comment:
This is correct. We have this space in translation and other strings have spaces